analyzers/tests/ITs.JsonParser.Test/Sarif.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace ITs.JsonParser.Test;
+
+internal static class Sarif
+{
+    internal static string CreateReport(string rootPath, string solution, string project, string tfm, params string[] issues)
+    {
+        var content = $$"""
+            {
+              "runs": [
+                {
+                  "results": [
+                    {{string.Join($",{Environment.NewLine}", issues)}}
```

Comment:
I prefer this one from SonarAnalyzer.Helpers
```suggestion
                    {{issues.JoinStr($",{Environment.NewLine}")}}
```